scripts/upgrade

```diff
@@ -9,7 +9,19 @@ source /usr/share/yunohost/helpers
 #=================================================
 ynh_script_progression "Stopping $app's systemd service..."
 
-ynh_systemctl --service=$app --action="stop" --log_path="systemd" --wait_until="Stopping Vaultwarden Server"
+ynh_systemctl --service=$app --action="stop" --log_path="systemd"
+
+#=================================================
+# MIGRATING DATABASE
+#=================================================
+
+# This migration should be done before the upgrade
+if psql -lqt | cut -d \| -f 1 | grep -qw "$db_name"; then
```

Comment:
I guess upgrade is stuck here?
I'd simply run if ynh_app_upgrading_from_version_before because it only has to run once